ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -50,6 +50,13 @@ tests:
     - chain: ipi-openstack-pre
     - ref: fips-check
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 72h
```

Comment:
Probably too frequent for a perf test.

Reply:
Yes, I realized it; I'm working to set up an appropriate interval.